versions.tf

```
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```

We can leave this at 0.12
```suggestion
  required_version = ">= 0.12"
```